CONTRIBUTING.md

```diff
@@ -1,21 +1,63 @@
-[Carlos Mastalli](https://cmastalli.github.io/) (University of Edinburgh): main developer and manager of the project
+## I. About
+Crocoddyl has been written in C++ for efficiency and uses the  Eigen  library  for  linear  algebra  routines.  It  comes  with Python bindings for easy prototyping. Crocoddyl is currently supported  for  most  Linux  distributions,  with  plans  to  release on Mac OS X and Windows. The project is fully open-source under  the  permissive  BSD-3-Clause  license  and  is  hosted on  GitHub.
 
-[Nicolas Mansard](http://projects.laas.fr/gepetto/index.php/Members/NicolasMansard) (LAAS-CNRS): project instructor
+## II. Design pattern
+Crocoddyl is designed using virtualization pattern for easy prototyping and yet efficient implementation. Virtualization have shown to be more efficient than static polymorphism through CRTP for system dynamics higher than 1. The real benefits of static polymorphism are in very small system (with dimension less than 6), however virtualization is still competitive.
```

Comment:
The second part of this paragraph seems to be over claiming. In general, static polymorphism by using CRTP and/or Variant are much more efficient.

Reply:
I benchmark it. For this particular case, it was worse for humanoid problems, same for quadruped and lightly better for unicycle.

Reply:
But when I've seen your bench, it was not correctly done I would say. I would remove this statement. Think to Eigen for instance ;)

Reply:
I have relaxed this claim by basically saying that it is competitive design. I will add in the next PR. Thanks for the feedback!